Wrappers/Python/cil/optimisation/operators/Operator.py

```diff
@@ -187,31 +187,34 @@ def dot_test(operator, domain_init=None, range_init=None, verbose=False, **kwarg
         :param domain_init: optional initialisation container in the operator domain 
         :returns: boolean, True if the test is passed.
         :param decimal: desired precision
-        :type decimal: int, optional, default 4
+        :type decimal: int, optional, default 4        

```

Comment:
doc string isn't up-to-date. But very useful to have tolerance!

Reply:
Fixed